plugins/scaffolder-react/api-report.md

```diff
@@ -352,6 +363,9 @@ export type StepperProps = {
   FormProps?: FormProps;
   initialState?: Record<string, JsonValue>;
   onComplete: (values: Record<string, JsonValue>) => Promise<void>;
+  ReviewStateWrapper?: (props: ReviewStateProps) => JSX.Element;
```

Comment:
Thinking this should be `ReviewStateComponent`

Reply:
done